.travis.yml

```diff
@@ -32,14 +32,10 @@ before_script:
   - sleep 3 # wait for xvfb to boot
   - yarn build-all
 script:
-  - yarn test-cli-formatting
-  - yarn test-launcher-formatting
-  - yarn lint
-  - yarn unit
-  - yarn closure
+  - yarn test-cli-formatting & yarn test-launcher-formatting & yarn lint & yarn unit & yarn closure & yarn compile-devtools & wait
```

Comment:
will a non-zero exit code in these background jobs trigger a failure?

By default I don't think it does, which is partially why i avoided doing it like so.